src/test/java/org/openlmis/buq/builder/SourceOfFundDataBuilder.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * This program is part of the OpenLMIS logistics management information system platform software.
+ * Copyright © 2017 VillageReach
+ *
+ * This program is free software: you can redistribute it and/or modify it under the terms
+ * of the GNU Affero General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details. You should have received a copy of
+ * the GNU Affero General Public License along with this program. If not, see
+ * http://www.gnu.org/licenses.  For additional information contact info@OpenLMIS.org.
+ */
+
+package org.openlmis.buq.builder;
+
+import java.util.UUID;
+import org.openlmis.buq.domain.sourceoffund.SourceOfFund;
+
+public class SourceOfFundDataBuilder {
+  private final UUID id = UUID.randomUUID();
+  private String name = "name";
+  private String description = "sof-description";
+
+  public SourceOfFundDataBuilder withName(String name) {
+    this.name = name;
+    return this;
+  }
+
+  public SourceOfFundDataBuilder withDescription(String description) {
+    this.description = description;
+    return this;
+  }
+
+  /**
+   * Builds new instance of SourceOfFund (with id field).
+   */
+  public SourceOfFund build() {
+    SourceOfFund sourceOfFund = buildAsNew();
+    sourceOfFund.setId(id);
+
+    return sourceOfFund;
+  }
+
+  /**
+   * Builds new instance of SourceOfFund as a new object (without id field).
+   */
+  public SourceOfFund buildAsNew() {
+    SourceOfFund sourceOfFund = new SourceOfFund();
+    sourceOfFund.setName(name);
+    sourceOfFund.setDescription(description);
+
+    return sourceOfFund;
+  }
+
+}
```

Comment:
I know it'd be probably useless for this flow, but lombok offers `@Builder` annotation. Maybe it'd be good to include it in the domain/dto?

Reply:
I wanted to stick to the OLMIS convention of DataBuilder classes in this case. As far as I know they are only used for testing purposes.